interface/themes/core.css

```diff
@@ -100,3 +100,7 @@ button[href*="deleter"]:before {
 .btn-graph::before {
     content: '\f201\00a0\00a0';
 }
+.page-header{
```

Comment:
Remove this from core, it's a theme-specific issue and recommend themes not even bothering with it